snap/snapcraft.yaml

```diff
@@ -84,30 +85,32 @@ parts:
   imagecraft:
     after: [imagecraft-libs, libgit2]
     source: .
-    plugin: python
+    plugin: uv
     build-attributes:
       - enable-patchelf
     build-packages:
-      - python3.12-dev
-      - python3-pip-whl
       - libffi-dev
       - cargo
       - pkg-config
       - git
-    python-packages:
-      - wheel
-      - pip
-      - setuptools
+    build-snaps:
+      - astral-uv
     build-environment:
-      - PARTS_PYTHON_VENV_ARGS: --system-site-packages
-      - PIP_NO_BINARY: ":all:"
+      - UV_COMPILE_BYTECODE: "1"
+      - XDG_CONFIG_DIRS: "${CRAFT_PART_BUILD}/xdg"
+      - MAKEOPTS: -j$(nproc --all)
     organize:
       bin/craftctl: libexec/imagecraft/craftctl
     override-build: |
-      craftctl default
+      # uv doesn't currently have an env var to force build from source,
```

Comment:
if you want to wait for `0.5.30` to hit stable you can get rid of this
@lengau might know when

Reply:
If it can hit stable before Friday then yes :)!

Reply:
🚢